tsconfig.eslint.json

```diff
@@ -0,0 +1,33 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "target": "es2022",
```

Comment:
Does it support node.js 14?

It must follow the fastify's lts matrix: https://fastify.dev/docs/latest/Reference/LTS/#schedule

Reply:
thanks for the review! I changed the `tsconfig` to be friendly with node.js 14 and I also included node 14 version to the CI checks